src/app/feature/consultant/components/consultant-header/consultant-header.component.html

```diff
@@ -2,6 +2,9 @@
   <ng-container *ngIf="consultant$ | async as consultant">
     <div class="picture-wrapper">
       <div class="button-container">
+        <button mat-fab class="export-button" style='background-color: green' (click)="exportProfile(consultant)">
```

Comment:
Please use a CSS class over inline styling. Inline styles are loaded first and should only be used for critical styling. Inline style could also lead to confusion due to breaking convention of keeping CSS in a separate file.